wiki/Medals/ru.md

```diff
@@ -1,58 +1,67 @@
----
-outdated_translation: true
-no_native_review: true
----
-
 # Медали
 
-Медали, ранее известные как *достижения*, — значки в профиле игрока, показывающие, что пользователь достиг необычной цели. В данный момент они разделены на 7 групп: [Beatmap Packs](#beatmap-packs), [Skill](#skill), [Dedication](#dedication), [Mod Introduction](#mod-introduction), [Hush-Hush](#hush-hush), Beatmap Spotlights и Seasonal Spotlights.
+Медали, ранее известные как *достижения*, — значки в профиле игрока, показывающие, что пользователь достиг необычной цели. В данный момент, существует 280 медалей для получения, разделённых на 8 групп: [Beatmap Packs](#beatmap-packs), [Skill](#skill), [Dedication](#dedication), [Mod Introduction](#mod-introduction), [Hush-Hush](#hush-hush), [Beatmap Spotlights](#beatmap-spotlights), [Seasonal Spotlights](#seasonal-spotlights), and [Beatmap Challenge Packs](#beatmap-challenge-packs).
 
 ## Beatmap Packs
 
-Медали группы **Beatmap Packs** могут быть получены за **игру на каждой карте до конца хотя бы на одной из сложностей** из [тематических сборок](https://osu.ppy.sh/beatmaps/packs?type=theme). Чтобы получить статус **`Пройдено`** в [списке карт сборки](https://osu.ppy.sh/beatmaps/packs?type=theme) и значок, вы обязаны играть в режиме [osu!](/wiki/Game_mode/osu!).
+Медали группы **Beatmap Packs** могут быть получены за **прохождение до конца хотя бы одной из сложностей на каждой карте** из [Тематических Сборок](https://osu.ppy.sh/beatmaps/packs?type=theme) или [Сборок Исполнителей/Альбомов](https://osu.ppy.sh/beatmaps/packs?type=artist). 
```

Comment:
```suggestion
Медали группы **Beatmap Packs** могут быть получены за **прохождение до конца хотя бы одной из сложностей на каждой карте** из [Тематических Сборок](https://osu.ppy.sh/beatmaps/packs?type=theme) или [Сборок Исполнителей/Альбомов](https://osu.ppy.sh/beatmaps/packs?type=artist).
```

Reply:
Don't seems to be exactly same as english (`These medals can be obtained by clearing every beatmap all the way through at least once in any difficulty within the corresponding beatmap pack.` removed ?). You have to stay as much as you can close to the en.md